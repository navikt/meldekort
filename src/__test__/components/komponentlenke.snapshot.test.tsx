import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper from '../utils/providerWrapper';
import Komponentlenke from '../../app/components/komponentlenke/komponentlenke';

it('Komponentlenke', () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <Komponentlenke lenketekst={'Test'} rute={'/testrute'} />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
