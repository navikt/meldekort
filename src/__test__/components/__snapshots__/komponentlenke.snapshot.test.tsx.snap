// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Komponentlenke 1`] = `
<a
  className="lenke"
  href="#"
  onClick={[Function]}
>
  Test
</a>
`;
